plugin/action/mask/mask.go

```diff
@@ -25,27 +27,39 @@ pipelines:
       - mask:
         re: "\b(\d{1,4})\D?(\d{1,4})\D?(\d{1,4})\D?(\d{1,4})\b"
         groups: [1,2,3]
+        metric_subsystem_name: "some_name"
     ...
 ```
 
 }*/
 
-const substitution = byte('*')
+const (
+	substitution = byte('*')
+
+	metricName = "mask_plugin"
+)
 
 type Plugin struct {
-	config     *Config
-	sourceBuf  []byte
-	maskBuf    []byte
-	valueNodes []*insaneJSON.Node
-	logger     *zap.SugaredLogger
+	config          *Config
+	sourceBuf       []byte
+	maskBuf         []byte
+	logMaskAppeared atomic.Bool
```

Comment:
bool? you only change this from constructor

Reply:
Bool'll cause race in case of panic during start of file.d and concurrent second `Start` call after recovering.